src/formats/Tinker.cpp

```diff
@@ -166,26 +166,25 @@ optional<uint64_t> TinkerFormat::forward() {
         return nullopt;
```

Comment:
Since forward is already making this check, I don't think it is worth catching the error here, since again it should only happen if the file was modified between the call to forward and the call to read_next